CLOUDFRONT.md

```diff
@@ -2,13 +2,11 @@
 1. Login to AWS console https://aws.amazon.com/console/
 2. Navigate to _Services -> S3_.
 3. Click _Create bucket_.
-4. Fill out bucket name, region and click _Create bucket_.
-5. Navigate to _My Security Credentials_.
-6. In the _Access keys_ section click on the _Create New Access Key_ button.
```

Comment:
I believe we still need to create an access key.

Reply:
I mean this doc is for setting up all the evn from scratch, so we need an access key. If someone already have a bucket or use IAM Role they can simply skip this step

Reply:
This text is confusing - S3 has no `Access keys` section and there's no reference to the other IAM steps required provide access to a bucket (but this is already detailed in `README.md`.

Reply:
There is a step before to navigate to _My Security Credentials_. I had a look at `README.md` and I couldn't find details there on how to create Access keys. Could you please point me on them?